README.mfem-pumi-install

```diff
@@ -0,0 +1,286 @@
+The following instructions are for building mfem with pumi on a SCOREC workstation.  Adjust
```

Comment:
I suggest we split this in two -- a brief description that goes into `INSTALL` and the more detailed instructions that can go on the webpage, e.g. as a link on http://mfem.org/building

Reply:
I have added a brief description in the `INSTALL` https://github.com/mfem/mfem/blob/pumi-dev/INSTALL#L352. @cwsmith what do you think about the webpage instructions? I see that this README explains installation on SCOREC machines and LLNL Vulcan. Should we keep it as it is for the website?